docs/api_reference/change-log.md

```diff
@@ -0,0 +1 @@
+# Change Log and Versioning
```

Comment:
We could already add the information we present when creating a tag on Github. Check the main Github page to have the info.